### DISCUSSION TITLE
Course progress exhaust and other reports not working

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Lern/Community#114] [Issues] [answered]
I have tried to create reports for a course by a course mentor in the dev env of sunbird and I am not able to see the reports in the UI
Course progress exhaust is in the submission state and the other reports like 
User Profile exhaust or Question set report I am not able to request also please let us know how to get this if any document is there please help us in understanding the same.
cc: @reshmi-nair @vinukumar-vs

**Comment by reshmi-nair** [comment, marked as answer]
@rajeevsathish Please run the jobs from Jenkins - Lern Dataproducts and check the logs in spark machine for errors- analytics/logs/lern-dataproducts/job-execution.log and analytics/scripts/logs/joblog.log. If any error, please share it.
